site/components/pages/Infinity/GetStartedModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import { Trans } from "@lingui/macro";
+import { urls } from "@klimadao/lib/constants";
+import { Text } from "@klimadao/lib/components";
+import { Modal } from "components/Modal";
+import hiker from "public/hiker.jpg";
+import net from "public/net.jpg";
+import building from "public/building.jpg";
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward";
+import * as styles from "./styles";
+
+export interface GetStardedModalProps {
+  showModal: boolean;
+  setShowModal: (value: boolean) => void;
+}
+
+const GetStartedModal = (props: GetStardedModalProps) => {
+  return (
+    <Modal
+      onToggleModal={() => props.setShowModal(false)}
+      showModal={props.showModal}
+      closeOnBackgroundClick={true}
+      title="Get Started"
+    >
+      <div className={styles.modalContainer}>
+        <Link href={urls.creolBusinessCalculator} passHref>
+          <div className={styles.modalButtonContainer}>
+            <div className="overlay" />
+            <Image
```

Comment:
Related to the comment from @0xAeterno about optimization
and also interesting for you @ladytrekker 

When we do layout="fill", but don't provide a `sizes` prop, Next will use the viewport width to decide which image to request. So actually for mobile devices, the requested image should be reasonably optimized thanks to [vercel](https://vercel.com/docs/concepts/image-optimization)

![mobile 375w](https://user-images.githubusercontent.com/88635679/188519228-3d3717af-1f53-48df-8e15-50e8e2198035.png)

In this case, that's good enough, because on mobile these images pretty much fill the viewport.
However for desktop users, there is some wasted bandwidth: the button max-width is 500px but we are requesting 2400px images!

So I think there are two solutions, and both should work fine
1. if this image is only used here, just export a 500px wide version
2. use the [sizes prop](https://nextjs.org/docs/api-reference/next/image#sizes) to specify that these images should not exceed 500px if the viewport is larger than 500px

Side note: even though vercel is optimizing for us it's probably still good to optimize images before adding them to our repo to reduce the overall size of our repository and reduce deploy times

Reply:
Thanks for the explanation. I will start using https://www.reduceimages.com/ to resize the images. These images are max 60kb after being resized.